build-setup-nolog.sh

```diff
@@ -171,10 +171,10 @@ if [ "$IS_LIBRARY" = false ]; then
 
     SKIP_TOOLCHAIN_ARG=""
     if [ "$SKIP_TOOLCHAIN" = true ]; then
-        SKIP_TOOLCHAIN_ARG="--skip-toolchain-extra"
+        SKIP_TOOLCHAIN_ARG="-s 3"
     fi
     # default to normal riscv-tools toolchain
-    ./build-setup.sh --skip-validate --skip-conda $SKIP_TOOLCHAIN_ARG
+    ./build-setup.sh --skip-validate --force $SKIP_TOOLCHAIN_ARG -s 1 -s 4 -s 5 -s 6 -s 7 -s 9
```

Comment:
```suggestion
    ./build-setup.sh --force $SKIP_TOOLCHAIN_ARG -s 1 -s 4 -s 5 -s 6 -s 7 -s 9
```

Reply:
Also, to match the old functionality, we should also skip 8 (Firemarshal submodule initialization - i.e. skip Linux checkout) (`-s 8`)

Reply:
How would you use firesim as top without firemarshal? Shouldn't this be initialized?

Reply:
In the past, this was not initialized by the setup script. I think to minimize the scope of this PR to just matching the old functionality you can not initialize it.

Reply:
Makes sense to me. I'm just a bit surprised that was the old behavior